README.rst

```diff
@@ -17,13 +17,16 @@ scikit-bio is currently in beta. We are very actively developing it, and **backw
 Installing
 ----------
 
+The recommended way to install scikit-bio is via the ``conda`` package manager available in `Anaconda <http://continuum.io/downloads>`_ or `miniconda <http://conda.pydata.org/miniconda.html>`_.
+
 To install the latest release of scikit-bio::
 
-    pip install scikit-bio
+    conda install -c https://conda.anaconda.org/biocore scikit-bio
```

Comment:
I'm using the longer form in case users have aliased `biocore` to something else in their condarc